ui/src/components/records/RecordFilePreview.svelte

```diff
@@ -8,35 +8,35 @@
 
     let previewPopup;
     let thumbUrl = "";
-    let originalUrl = "";
-
-    $: hasPreview = CommonHelper.hasImageExtension(filename);
-
-    $: if (hasPreview) {
-        originalUrl = ApiClient.getFileUrl(record, `${filename}`);
-    }
+    let originalUrl = ApiClient.getFileUrl(record, `${filename}`);
 
+    $: type = CommonHelper.getFileType(filename);
+    $: hasPreview = /^(image|pdf)$/.test(type);
```

Comment:
I don't think the regex is needed in this case. It would be more readable with direct checks or something like `["image", "pdf"].includes(type)`.